sa/sysvars.go

```diff
@@ -0,0 +1,350 @@
+package sa
+
+import (
+	"fmt"
+	"regexp"
+)
+
+var checkStringQuoteRE = regexp.MustCompile(`^'[0-9A-Za-z_\-=:]+'$`)
+var checkIntRE = regexp.MustCompile(`^\d+$`)
+var checkImproperIntRE = regexp.MustCompile(`^'\d+'$`)
+var checkFloatOrIntRE = regexp.MustCompile(`^\d+(\.\d+)?$`)
```

Comment:
nit: you use "Numeric" on the name of the map below; I'd use "checkNumericRE" here too.